techniques/general/G87.html

```diff
@@ -207,39 +183,35 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
          </ul>
       
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G93">G93: Providing open (always visible) captions</a></li>
       <li><a href="../smil/SM11">SM11: Providing captions through synchronized text streams in SMIL 1.0</a></li>
       <li><a href="../smil/SM12">SM12: Providing captions through synchronized text streams in SMIL 2.0</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Turn on the closed caption feature of the media player</li>
-            <li>View the synchronized media content</li>
-            <li>Check that captions (of all dialogue and important sounds) are
-                  visible and in the <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-human-language-s" target="terms">human language</a> of the video</li>
+            <li>メディアプレーヤーのクロ―ズドキャプション機能をオンにする。</li>
+            <li>同期したメディアのコンテンツを見る。</li>
+            <li>(発話内容及び音の伝えている情報全ての) キャプションが、動画の<a href="https://waic.jp/translations/WCAG22/#dfn-human-language-s" target="terms">自然言語</a> で表示されていることをチェックする。</li>
```

Comment:
important が訳出されていないので調整しました。
```suggestion
            <li>(発話内容及び重要な音の伝えている情報全ての) キャプションが、動画の<a href="https://waic.jp/translations/WCAG22/#dfn-human-language-s" target="terms">自然言語</a> で表示されていることをチェックする。</li>
```